Resources/MultiplayerGameLobby.ini

```diff
@@ -461,7 +461,7 @@ $Y=getBottom(chkNoSpy) + GAME_OPTION_GAP
 [chkInstantDisconnectKick]
 SpawnIniOption=ReconnectTimeout
 DefaultValue=False
-EnabledSpawnIniValue=0
```

Comment:
What is this value? Is this how long we have to wait for someone to reconnect? Isn't it currently 30 seconds?

Reply:
it's frames I believe, @GrantBartlett may confirm

The default being 2400 @devo1929

Reply:
When the option is enabled the DC screen will be skipped entirely